integration_tests/src/main/python/explain_test.py

```diff
@@ -91,3 +91,53 @@ def do_explain(spark):
 
     with_cpu_session(do_explain)
 
+@allow_non_gpu(any=True)
+def test_explain_bucketed_scan(spark_tmp_table_factory):
+    """
+    https://github.com/NVIDIA/spark-rapids/issues/3952
```

Comment:
A text explanation would be good to be able to get a high level idea of what is happening.

Reply:
Thank you for the feedback! I will add comments.